src/psyclone/tests/psyir/nodes/omp_directives_test.py

```diff
@@ -49,7 +49,9 @@
 from psyclone.psyir.nodes import OMPDoDirective, OMPParallelDirective, \
     OMPParallelDoDirective, OMPMasterDirective, OMPTaskloopDirective, \
     OMPTaskwaitDirective, OMPTargetDirective, OMPLoopDirective, Schedule, \
-    Return, OMPSingleDirective, Loop, Literal, Routine, Assignment, Reference
+    Return, OMPSingleDirective, Loop, Literal, Routine, Assignment,\
+    Reference, OMPNowaitClause, OMPGrainsizeClause, OMPNumTasksClause,\
+    OMPNogroupClause
```

Comment:
Can you test the node.clauses property in some of this directives (we are currently missing the coverage of the clauses property with a StandaloneDirective which is just a check that it returns an empty string)

Reply:
Added to OMPTaskloopDirective tests.